### ISSUE TITLE
Train more than 100 epochs on VOC 0712 but not convergent

### ISSUE THREAD

**Issue by unlabeledData** [Stick-To/CenterNet-tensorflow#26]
I trained on voc 0712 with 15000+ images using your code.
After trained more than 100 epochs, the mean loss on train set is 4.27.
Than I test this model on voc 07 test set, and I found that the model is not convergent.
It can not detect any objects.

All the hyper parameters are as same as your setting.

**Comment by codingali1** [comment]
Hello，I meet the same problem，my loss is around 3 after 180 epoch，and I got the poor result。Could you please explain the what the method to resolve this problem？

Many thanks！！！
